### PR TITLE
Document the .NET solutions for WireMock

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Documents existing WireMock related projects for the .NET ecosystem. I put WireMock UI to that page too, until we have a strategy on the official UI, if any.

CC @StefH @cezarypiatek @juniorgasparotto @alexvictoor 

## Preview

localhost_4000_docs_solutions_dotnet_

### CODE REVIEW

**Comment by cezarypiatek** [comment]
Not sure what you mean by "within .NET test frameworks". ".NET Framework" and "dotnet core" are not "test frameworks".
I would skip that phrase to avoid confusion.

image


The rest looks good to me.

**Comment by oleg-nenashev** [comment]
Agreed. I meant NUnit & Co, but indeed there's possible confusion

File: _docs/solutions/dotnet.md [hunk 1]
```diff
@@ -4,5 +4,64 @@ title: "WireMock and .NET"
...
+
+## Mock4Net
+
+A tiny mock HTTP server for .NET, inspired by WireMock.
+A fluent API allows to specify the behavior of the server and hence easily stub and mock
+web services and REST resources.
+
+References:
+
+- [GitHub Repository](https://github.com/alexvictoor/mock4net)
```
Comment (oleg-nenashev) [inline _docs/solutions/dotnet.md:37]:
```suggestion
```

File: _docs/solutions/dotnet.md [hunk 2]
```diff
@@ -4,5 +4,64 @@ title: "WireMock and .NET"
...
-coming-soon: true
 ---
+
+
+## WireMock.Net
+
+A .NET implementation of a API mock server in C# based on
+[mock4net](https://github.com/alexvictoor/mock4net)
+It mimics the functionality from [WireMock](https://github.com/wiremock/wiremock) implemented in Java.
+WireMock.NET can be used with all .NET based languages within .NET test frameworks,
```
Comment (oleg-nenashev) [inline _docs/solutions/dotnet.md:15]:
```suggestion
WireMock.NET can be used with all .NET based languages,
```

**Pull request merged**